feat: wallet dialog box added_CU-85zt5ydv2

subinsk opened the pull request.

irshadCypherock committed:
ci: removed version ci

subinsk committed:
feat: wallet dialog box added

rohanagarwal94 commented:
Task linked: [PRF-4784 UI for Create Wallet and Import Wallet Page](https://app.clickup.com/t/9002019994/PRF-4784)

irshadCypherock committed:
build(fix): mac build issue (#27)

subinsk committed:
style: added golden question mark and list container

Commit:
feat: card tap check, auth and congratulations screen ui (#26)

* feat: card tap check and auth ui

Squashed commit of the following:

commit 4a9e96bbe8d471d4fe26428083183ef31ac08c76
Author: Tejasv Sharma <tejasvonly@gmail.com>
Date:   Wed May 31 12:18:17 2023 +0530

    refactor: use keyboardevent instead of custom type

commit 41e18e72cd0d557fa675ca77a9e197e7f53952d4
Author: Tejasv Sharma <tejasvonly@gmail.com>
Date:   Tue May 30 22:41:45 2023 +0530

    refactor: review changes

commit eb33e9ccbe2e4db228849f598e3c5ab849b378a0
Merge: 514bf48 4ffe46b
Author: Tejasv Sharma <tejasvonly@gmail.com>
Date:   Tue May 30 22:41:33 2023 +0530

    Merge remote-tracking branch 'origin/develop' into feat/card-tap-check

commit 514bf48af9155a8ad97bed35d4480d10e77477a9
Author: Tejasv Sharma <tejasvonly@gmail.com>
Date:   Tue May 30 18:35:14 2023 +0530

    fix: use correct api

commit 01da36ecafff73ae87e7cee56610a80eef88d200
Merge: ae6a074 9493215
Author: TejasvCypherock <107414753+TejasvCypherock@users.noreply.github.com>
Date:   Tue May 30 17:57:49 2023 +0530

    Merge branch 'develop' into feat/card-tap-check

commit ae6a074793065decb95ea6e15fc2b5cb070e771f
Merge: 8095ce7 b16e4b0
Author: Tejasv Sharma <tejasvonly@gmail.com>
Date:   Tue May 30 17:52:54 2023 +0530

    Merge remote-tracking branch 'origin/main' into feat/card-tap-check

commit 8095ce7df3c113526ccdd45905bccaaeeb6e61f5
Author: Tejasv Sharma <tejasvonly@gmail.com>
Date:   Tue May 30 17:47:45 2023 +0530

    feat: added card auth and congratulations screen

commit f537f7ea3533640544c71e6c10ff88833a73dad7
Author: Tejasv Sharma <tejasvonly@gmail.com>
Date:   Sat May 27 18:42:57 2023 +0530

    feat: add card tap check initial page

commit e2ea858ea73b0c9d80fca235caa49827b43c96df
Author: Tejasv Sharma <tejasvonly@gmail.com>
Date:   Sat May 27 18:42:00 2023 +0530

    fix: update aside container's image handling

commit a2c5b826ed4aaf7bbc860c9fc5bb6d72b19a6c42
Merge: c26cdd8 6bb173c
Author: Tejasv Sharma <tejasvonly@gmail.com>
Date:   Sat May 27 17:39:16 2023 +0530

    Merge remote-tracking branch 'origin/main' into feat/card-tap-check

commit c26cdd83b0902ef313a477851ae3cc7df2aa303a
Author: Tejasv Sharma <tejasvonly@gmail.com>
Date:   Fri May 26 17:14:35 2023 +0530

    fix: use correct page to redirect after auth

commit a587ec561e87778855ab3fae38c885b85819d64b
Author: Tejasv Sharma <tejasvonly@gmail.com>
Date:   Fri May 26 16:58:03 2023 +0530

    refactor: review changes

commit 4a431422572935f1628b22eb5c5629083f80b4ab
Author: Tejasv Sharma <tejasvonly@gmail.com>
Date:   Thu May 25 19:09:41 2023 +0530

    refactor: rename translate variable

commit 38a76c580eec578ed33becc995299eb176f045b8
Author: Tejasv Sharma <tejasvonly@gmail.com>
Date:   Thu May 25 17:54:35 2023 +0530

    chore: update package

commit eb6af925810b78fc5e6a9ad8ef8a8ea9da015831
Merge: 38a5f1a eb60a7e
Author: Tejasv Sharma <tejasvonly@gmail.com>
Date:   Thu May 25 17:45:48 2023 +0530

    Merge remote-tracking branch 'origin/main' into joystick-training

commit 38a5f1a50d6d7552e6e4015f230ff80721fc30db
Author: Tejasv Sharma <tejasvonly@gmail.com>
Date:   Thu May 25 17:41:47 2023 +0530

    chore: update icons and submodule

commit 028328714839f5a22dd5c9c5fca95e104644db82
Author: Tejasv Sharma <tejasvonly@gmail.com>
Date:   Thu May 25 14:09:48 2023 +0530

    feat: joystick training functionality

commit 3b0805dd4f1aebab72bd5eb81c4538b4eab10946
Author: Tejasv Sharma <tejasvonly@gmail.com>
Date:   Thu May 25 13:05:03 2023 +0530

    feat: joystick training implementation

* chore: update changeset

* refactor: update return type for usestatewithfinality hook

---------

Co-authored-by: Tejasv Sharma <tejasvonly@gmail.com>

Commit:
feat: loading screen ui (#28)

* feat: loading screen ui

* chore: update changeset

* fix: made default version empty

---------

Co-authored-by: Tejasv Sharma <tejasvonly@gmail.com>

subinsk committed:
feat: list items updated

irshadCypherock committed:
feat: disconnect and abort device operation on quit (#30)

* feat: disconnect and abort device operation on quit

* chore: updated sdk version

irshadCypherock committed:
feat: i18n store to support multiple lang (#29)

* feat: i18n store to support multiple lang

* refactor: replaced redux connect with hooks

* refactor: onboarding layout

* refactor: onboarding layout

malviyagaurav committed:
test: playwright integration (#33)

* feat: playwright integration

* feat: 2 test case added

* feat: device auth and joystick training test case added

* feat: device disconnection test case added

* feat: playwright merge with developbranch

* fix: resolve merge conflicts

* test: pr changes resolution

* test: remaining changes

* test: ci changes

* chore: updated pnpm lock

* test: ci changes

* test(ci): fix e2e tests

* test(ci): fix e2e tests

* test(ci): fix test ci

* test(ci): separated e2e ci

* test(ci): fix e2e ci

* test(ci): fix e2e ci

* test(ci): fix e2e ci

* test(ci): fix e2e ci

* test: playwright integration

* test: updated pacage.json

* test: tsconfig.eslint.json update

* Revert "test: tsconfig.eslint.json update"

This reverts commit 85a6491c62ca092d16216292dfb9cc1f312af8af.

* test: revert few changes in tsconfig.eslint.json

---------

Co-authored-by: Gaurav Malviya <techymalviya@gmail.com>
Co-authored-by: Md Irshad Ansari <irshad@cypherock.com>

subinsk committed:
feat: updated for minimum screen

subinsk committed:
feat: language added
